gke-platform/modules/kuberay/kuberay-operator-autopilot-values.yaml

```diff
@@ -4,9 +4,9 @@
 
 image:
   repository: kuberay/operator
-  tag: nightly
+  tag: v0.6.0
```

Comment:
any chance we can push this fix in a separate PR? I think `main` might be broken at the moment using nightly builds?

Reply:
https://github.com/GoogleCloudPlatform/ai-on-gke/pull/81